BlueDBOnDisk/src/main/java/io/bluedb/disk/recovery/IndividualChange.java

```diff
@@ -0,0 +1,59 @@
+package io.bluedb.disk.recovery;
+
+import java.io.Serializable;
+
+import io.bluedb.api.keys.BlueKey;
+import io.bluedb.disk.segment.Range;
+import io.bluedb.disk.serialization.BlueEntity;
+
+public class IndividualChange <T extends Serializable> implements Serializable, Comparable<IndividualChange<T>> {
+
+	private static final long serialVersionUID = 1L;
+
+	private final BlueKey key;
+	private final T oldValue;
+	private final T newValue;
+
+	public static <T extends Serializable> IndividualChange<T> insert(BlueKey key, T value) {
+		return new IndividualChange<T>(key, null, value);
+	}
+
+	public IndividualChange(BlueKey key, T oldValue, T newValue) {

```

Comment:
Looks like it'll be pretty easy to make a batch update or batch delete using this class. Is that accurate? I like it!

Reply:
Yes, by design